### PR TITLE
N2kyni contacts

### PR DESCRIPTION
**Pull request by timoasumaniemi**
Remove links from text

### CODE REVIEW

File: src/common/components/Footer/Footer.js [hunk 1]
```diff
@@ -19,17 +19,19 @@ export const Footer = () => {
             <ul className="list-unstyled">
               <li>
                 <a href="#" className="footer-link">
```
Comment (timoasumaniemi) [inline src/common/components/Footer/Footer.js:21]: Remove links from text

**Review by timoasumaniemi — COMMENTED** [review]
Fix the commented links

File: src/common/components/Footer/Footer.js (hunk 1 above)
Comment (NicoK81) [reply src/common/components/Footer/Footer.js]: all good

**Review by timoasumaniemi — COMMENTED** [review]
good to go

File: src/common/components/Footer/Footer.js [hunk 2]
```diff
@@ -18,18 +18,20 @@ export const Footer = () => {
...
-                <a href="#" className="footer-link">
-                  Lorem ipsum dolor
+                <a>
+                Daavidin divari on vuodesta 1929 toiminut kauppa.
                 </a>
               </li>
               <li>
-                <a href="#" className="footer-link">
-                  Sit amet
+                <a>
```
Comment (jorijoo) [inline src/common/components/Footer/Footer.js:26]: Change to divs instead of as

File: src/common/components/Footer/Footer.js [hunk 3]
```diff
@@ -18,18 +18,20 @@ export const Footer = () => {
...
+                Daavidin divariin on tervetullut koko perhe.
                 </a>
               </li>
               <li>
-                <a href="#" className="footer-link">
-                  Consectetur
+                <a>
```
Comment (jorijoo) [inline src/common/components/Footer/Footer.js:31]: Div instead of a

**Review by jorijoo — CHANGES_REQUESTED** [review]
Check the comments in Footer.js

**Review by timoasumaniemi — COMMENTED** [review]
I think the `daavidin_divari.jpg` file can be removed from the `public` folder. 
Otherwise should be good for merge.

**Review by timoasumaniemi — COMMENTED** [review]
Gg

**Review by jorijoo — APPROVED** [review]
All the changes are as requested, good job!

**Pull request merged**